source/_integrations/roon.markdown

```diff
@@ -15,16 +15,11 @@ The Roon integration allows you to control [RoonLabs](https://roonlabs.com/) mus
 
 This integration uses Roon Core, a Roon application that runs on a machine on your network. Via Roon Core, Home Assistant can control all the Roon music players on your network.
 
-To integrate with Roon, you need to provide Home Assistant with the Hostname or IP address of the machine that runs your Roon Core, and then authorize Home Assistant in the Roon application.
-
-If you use an IP address, please assign a static IP address to the machine that runs Roon Core. This ensures that it won't change IP addresses, so you won't have to change the configuration in Home Assistant if it reboots and changes IP address. See your router's manual for details on how to set this up.
-
 ## Configuration
 
-You need the Hostname or IP address of the machine that runs your Roon Core. This might be a machine name (which can be followed by `.local`, e.g., `myserver.local`) or can be an IP address.
-
 1. From the Home Assistant front-end, navigate to **Configuration** then **Integrations**. Under **Set up a new integration** locate 'Roon' and click **Configure**.
-2. Enter the `Hostname` or `IP address` for the Roon Core machine and click **Submit**.
```

Comment:
These changes are not related to the parent PR.
If anything, this whole block should be replaced by including the `integrations/config_flow.md` in a separate PR.

Reply:
I've raised a new PR against current for the discovery stuff:  https://github.com/home-assistant/home-assistant.io/pull/17599

There is a roon specific authorisation step in the middle - so I couldn't use the include.

Will removes that stuff from this PR.

Reply:
Will close this and replace with https://github.com/home-assistant/home-assistant.io/pull/17600

Reply:
That specific step in the middle should not be needed to be documented, as it should be clear what one should do from the UI directly.